.github/workflows/static_code_checks.yaml

```diff
@@ -23,5 +23,7 @@ jobs:
       - name: precommit checks
         run: |
           pip install --upgrade pip
-          pip install -r requirements.txt
+          pip install $(grep -v '^torchdata\|^torchtext\|^torcheval' requirements.txt)
           pre-commit run --all-files
+      - name: Clearing pip for space
+        run: pip uninstall -y -r requirements.txt
```

Comment:
For whatever reason, we started running out of disk space to save the pip cache on the github runner. This removes the site-packages to save a bit of space and allow the caching to complete